roofit/roofitcore/inc/RooArgSet.h

```diff
@@ -74,6 +74,7 @@ class RooArgSet : public RooAbsCollection {
   }
 
   RooArgSet(const RooArgSet& other, const char *name="");
+  RooArgSet(RooArgSet&& other);
```

Comment:
You could do it for RooArgList as well to have it for all three.

Reply:
Done!